Doc/library/functions.rst

```diff
@@ -1007,18 +1018,19 @@ are always available.  They are listed here in alphabetical order.
       The contents of this dictionary should not be modified; changes may not
       affect the values of local and free variables used by the interpreter.
 
-.. function:: map(function, iterable, ...)
+.. function:: map(function, *iterables)
 
-   Return an iterator that applies *function* to every item of *iterable*,
-   yielding the results.  If additional *iterable* arguments are passed,
+   Return an iterator that applies *function* to every item of *iterables*,
+   yielding the results.  If additional *iterables* arguments are passed,
```

Comment:
I don't think the new wording is true, e.g.:

```pycon
>>> from operator import add
>>> for x in map(add, [1, 2, 3, 4, 5], [10, 20, 30, 40, 50]):
>>>     print(x)
>>> 
11
22
33
44
55
```

So it doesn't apply `function` to every item of `iterables`, but to every item of `zip(*iterables)`.

Reply:
you're right, we have to rephrase it..

Reply:
What about 7aca0e534a16c541f1f6fc7ba94b967da0f1d5a1 ? `map(int)` is not valid anyway.

Reply:
(I move the / in the following commit as `iterable` can't be given by name)